index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Mobile First</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" class="logo">Nasir</a>
+
+        <nav class="button">
+            <img src="images/Union.png" alt="button">
+        </nav>
+    </header>
+
+    <main id="main-section">
+        <section class="about">
+
+
+            <h1>Hi, I am Nasir.<br>
+                Glad to see <br>
+                you!</h1>
+
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact
+                me.
+            </p>
+
+            <a href="#">LET'S CONNECT</a>
+            <ul class="socialmedia">
+                <li class="twitter"><a href="#"><img src="images/twitter.png" alt="twitterlogo"></a></li>
+                <li class="linkedin"><a href="#"><img src="images/linkedIn.png" alt="linkedinlogo"></a></li>
+                <li class="medium"><a href="#"><img src="images/medium.png" alt="mediumlogo"></a></li>
+                <li class="github"><a href="#"><img src="images/github.png" alt="githublogo"></a></li>
+                <li class="angellist"><a href="#"><img src="images/angellist.png" alt="angellistlogo"></a></li>
+            </ul>
+
+        </section>
+    </main>
+    <!-- CARDS SECTION -->
+    <section class="work-section">
+
+        <!-- CARD 1 -->
+        <div class="card">
+
+            <div class="cardimage">
+                <img src="images/Snapshoot(1).png" alt="">
+            </div>
+
+            <div class="workdescription">
+
+                <h2>Tonic</h2>
+
+                <div class="details">
+                    <span id="canopy">CANOPY</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair" id="wider">Back End Dev</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair">2015</span>
+                </div>
+
+                <p>A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li class="one">HTML</li>
+                    <li class="two">CSS</li>
+                    <li class="three">Javascript</li>
+                </ul>
+
+                <button>See project</button>
```

Comment:
- [ ] Kindly add a hover and pressed state to all your buttons to match the Figma template. You can use the `:hover` and `:active` pseudoclasses for this.
![image](https://user-images.githubusercontent.com/94551484/235258797-93851e6d-1d24-420b-b790-7ba70c9b51e8.png)

Reply:
@ The visuals provided here are quite helpful. Much appreciated.